sympy/matrices/expressions/tests/test_matmul.py

```diff
@@ -119,3 +119,9 @@ def test_collapse_MatrixBase():
 
 def test_refine():
     assert refine(C*C.T*D, Q.orthogonal(C)).doit() == D
+
+def test_issue_9503():
+    from sympy import I
+    a = symbols('a')
+    M = Matrix([[1, 2 + I], [3, 4]])
+    assert transpose(MatMul(a, M)) == MatMul(transpose(a), transpose(M))
```

Comment:
> Plain a instead of transpose(a) would seem more suitable for a scalar a

```
>>> from sympy.matrices.expressions.transpose import transpose as mat_trans
>>> from sympy.functions.elementary.complexes import transpose as func_trans
>>> a = Symbol('a')
>>> mat_trans(a) == a
False
>>> func_trans(a) == a
False
```
Why do you expect any of these print plain `a` instead of `mat_trans(a)` or `func_trans(a)` ?